client/blocks/author-selector/index.jsx

```diff
@@ -95,7 +95,7 @@ const SwitcherShell = React.createClass( {
 					{ ( this.props.fetchOptions.search || users.length > 10 ) &&
 						<Search
 							onSearch={ this._onSearch }
-							placeholder={ this.translate( 'Find Author…', { context: 'search label' } ) }
+							placeholder={ this.translate( 'Find Author...', { context: 'search label' } ) }
```

Comment:
Warning: 27 translations will be lost with this change.
[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/tests/localci-test-branch)